.github/workflows/python-wrapper-wheel.yml

```diff
@@ -0,0 +1,89 @@
+# Releases a python-wrapper wheel: for an ecbuild-based compiled library,
+# runs the compilation and bundles all artifacts (libs, includes, ...) into
+# a binary wheel (manylinux/macos) along with findlibs-compatible instructions
+# about dependencies
+# Assumes that the project has the `python_wrapper` directory which configures
+# this action
+
+on:
+  workflow_call:
+    inputs:
+      use_test_pypi:
+        description: Use test pypi instead of the regular one
+        required: false
+        type: boolean
+        default: false # TODO actually use this
+
+jobs:
+  linux-wheel:
+    name: Build manylinux_2_28
+    strategy:
+      matrix:  
+        # TODO convert this to be matrix-friendly. Note it's a bit tricky since
+        # we'd ideally not reexecute the compile step multiple times, but it
+        # (non-essentially) depends on a matrix-based step. Add more pythons, possibly
+        # more manylinuxes
+        python_version: ["3.11", "3.13"]
```

Comment:
I'd add more Pythons from the start (3.9 to 3.13) because it will be a pain to add them later if we've already uploaded wheels for some Python versions and not others (I know from experience!)

Reply:
sure, I just didn't want the initial test to run that long :), I'll update pre-merge. Also, note each entry in the matrix still requires the full compilation (redundantly...)